zulip_bots/zulip_bots/bots/googlesearch/googlesearch.py

```diff
@@ -47,8 +50,7 @@ def get_google_result(search_keywords):
             logging.exception(t_err)
             return "Error: Google search function failed. {}.".format(t_err)
         except Exception as e:
-            logging.exception(e)
-            return 'Error: Search failed. {}.'.format(e)
+            return 'Error: Search failed.'
```

Comment:
What was the reasoning behind this change? We usually prefer to see the error log for easy debugging.

Reply:
The errors which were thrown were just empty.